src/disco/containers/DiscoPane.js

```diff
@@ -33,6 +33,8 @@ export class DiscoPaneBase extends React.Component {
     i18n: PropTypes.object.isRequired,
     location: PropTypes.object.isRequired,
     mozAddonManager: PropTypes.object,
+    params: PropTypes.object.isRequired,
+    platform: PropTypes.string.isRequired,
```

Comment:
Aaaah I see. I think you dont need to do that. Instead, rely on `params.platform` in the code.